src/components/screens/delegates/delegates.test.js

```diff
@@ -48,17 +37,8 @@ describe('Delegates', () => {
     loadDelegates: jest.fn(),
     loadVotes: jest.fn(),
   };
-  const options = {
-    context: { store, history, i18n },
-    childContextTypes: {
-      store: PropTypes.object.isRequired,
-      history: PropTypes.object.isRequired,
-      i18n: PropTypes.object.isRequired,
-    },
-  };
-
   it('should allow to enable and disable voting mode', () => {
-    wrapper = mount(<Router><Delegates {...props} /></Router>, options);
+    wrapper = mount(<Router><Delegates {...props} /></Router>);
```

Comment:
I think is a good practice to create a function to return the wrapper and you can pass the props that you want to the function, something like this in the describe section.

`
const setup = (props) => mount(<Router><Delegates {...props} /></Router>);
`

Then in each it you can called like.

`
const wrappere = setup(newProps);
`

So we can avoid to repeat always the same code and do the same for all the files where we duplicate code like this.

Reply:
I like this suggestion and agree that we could use `const wrapper = setup(props)` in all tests that use `mount` .

However, it's IMO something outside the scope of this PR as there are a lot of tests that should be updated to comply with that convention. 

I suggest you to bring this topic up in the next technical meeting so that we can agree on it with the whole team.